app.js

```diff
@@ -1,12 +1,14 @@
 const express = require("express");
 const app = express();
-const { getCategories, getReviews } = require("./controller/controller.js");
+const { getCategories, getReviews, getUsers } = require("./controller/controller.js");
```

Comment:
I would suggest maybe separating out your controller to different files for different data sets e.g categoriesController, reviewsController and so on